### ISSUE TITLE
keymap conflict with multi cursor

### ISSUE THREAD

**Issue by dotku** [facebooknuclide/hyperclick#40]
cmd+click is used for multiple cursor select, conflict with this package.

**Comment by jackkeller** [comment]
Yes, I've experienced this too, can we map this to another thing in our `keymap.cson`?

**Comment by payne8** [comment]
I have uninstalled hyperclick and the multi-cursor mapping was not restored. How do I bring it back?

**Comment by WLyKan** [comment]
see the atom-ide-ui setting
![image](https://user-images.githubusercontent.com/2155093/34811496-210d0ed4-f6dc-11e7-96f2-04d6183def86.png)

=============
it's really confusing, use `alt + click` is a better solution for hyperclick.
`cmd + click` is for multi selector is a useful keybinding for all atom users.
